src/frontpage/components/Articles/MainArticle.tsx

```diff
@@ -1,12 +1,11 @@
 import { IArticle } from 'articles/models/Article';
 import ResponsiveImage from 'common/components/ResponsiveImage';
-import { DOMAIN } from 'common/constants/endpoints';
 import React from 'react';
 import style from './articles.less';
 
-const MainArticle = ({ absolute_url, heading, image, ingress }: IArticle) => {
+const MainArticle = ({ heading, image, ingress, id }: IArticle) => {
   return (
-    <a href={DOMAIN + absolute_url}>
+    <a href={'articles/' + id}>
```

Comment:
In this case you can import `routes` from `ArticlesRouter`, and use `routes.detail`

Reply:
I added it now, using `<Link>` instead of  `<a>`, but is the only way to add the `id` to it by slicing the string?